stable_plugins/visualization/file_types/qasm_visualization.py

```diff
@@ -196,10 +196,10 @@ def get_circuit_image(data: Mapping):
     url_hash = hashlib.sha256(url.encode("utf-8")).hexdigest()
     image = DataBlob.get_value(QasmVisualization.instance.identifier, url_hash, None)
     if image is None:
-        response = Response("Image not yet created!", HTTPStatus.SERVICE_UNAVAILABLE)
-        response.headers["Retry-After"] = 5
-        if not PluginState.get_value(
-            QasmVisualization.instance.identifier, url_hash, None
+        if not (
```

Comment:
To avoid such nesting, use a standard variable before the if (however this is still ok, so no need to change)